README.md

```diff
@@ -216,6 +216,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Mini Golf](http://drawvr.com/minigolf/) (by [@drawvr]) - 9 Holes
 - [Welcome to A-Frame](http://drawvr.com/welcome/) (by [@drawvr]) - Quick introduction to A-Frame
 - [Photo Sphere Diver](https://www.ideaspacevr.org/themes/web-vr-photo-sphere-diver) (by [IdeaSpace](https://www.ideaspacevr.org/)) - Dive into your photo spheres
+- [Sketch VR](http://uniform-futures.github.io/sketch-vr/) (by [@uniformfutures]) -  See your paper doodle in VR
```

Comment:
Thanks for the corrections. It should be fine now :)


----------------------------------------

* Leonardo Amico *


* Designer & Engineer*

* W. orgonomyproductions.info <http://orgonomyproductions.info/>*

* T. @orgonomyprod <https://twitter.com/orgonomyprod>** S. leonardo.amico *
 M. *+44 (0) 790 376 8822*
----------------------------------------

2016-06-01 22:06 GMT+01:00 chris van wiemeersch <notifications@github.com>:

> In README.md
> <https://github.com/aframevr/awesome-aframe/pull/79#discussion_r65442695>:
>
> > @@ -216,6 +216,7 @@ Use React with A-Frame. Share your A-Frame React Components!
> >  - [Mini Golf](http://drawvr.com/minigolf/) (by [@drawvr]) - 9 Holes
> >  - [Welcome to A-Frame](http://drawvr.com/welcome/) (by [@drawvr]) - Quick introduction to A-Frame
> >  - [Photo Sphere Diver](https://www.ideaspacevr.org/themes/web-vr-photo-sphere-diver) (by [IdeaSpace](https://www.ideaspacevr.org/)) - Dive into your photo spheres
> > +- [Sketch VR](http://uniform-futures.github.io/sketch-vr/) (by [@uniformfutures]) -  See your paper doodle in VR
>
> perhaps rename summary if you'd like to Doodle on paper, take a picture,
> and view it in VR?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/aframevr/awesome-aframe/pull/79/files/7b1092afbd85aa5a30975c1918cbcde504938687#r65442695>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABmRaLmcrPXfVKjxBujnDwlz90Qyp1aGks5qHfRdgaJpZM4IrhF0>
> .
>